tools/base/fpgad/config_file.c

```diff
@@ -30,7 +30,257 @@
 
 #include "config_file.h"
 
+#include <json-c/json.h>
 
+#ifdef LOG
+#undef LOG
+#endif
+#define LOG(format, ...) \
+log_printf("cfg: " format, ##__VA_ARGS__)
 
+int cfg_find_config_file(struct fpgad_config *c)
```

Comment:
Would it be worth defining a macro that checks a path and copies it to config structure?

Reply:
sure